Autopilot/Path Manager/PathManager.c

```diff
@@ -640,18 +640,15 @@ unsigned int insertPathNode(PathData* node, unsigned int previousID, unsigned in
 }
 
 void copyGPSData(){
-    if (newGPSDataAvailable && gpsErrorCheck(gpsData.latitude, gpsData.longitude)){
-        newGPSDataAvailable = 0;
-        
-        interchip_send_buffer.pm_data.time = gpsData.time;
-        interchip_send_buffer.pm_data.longitude = gpsData.longitude;
-        interchip_send_buffer.pm_data.latitude = gpsData.latitude;
-        interchip_send_buffer.pm_data.heading = gpsData.heading;
-        interchip_send_buffer.pm_data.speed = gpsData.speed;
-        interchip_send_buffer.pm_data.satellites = (char)gpsData.satellites;
-        interchip_send_buffer.pm_data.positionFix = (char)gpsData.positionFix;
-        checkForFirstGPSLock();
-    }
+    interchip_send_buffer.pm_data.time = gps_data.utc_time;
+    interchip_send_buffer.pm_data.longitude = gps_data.longitude;
+    interchip_send_buffer.pm_data.latitude = gps_data.latitude;
+    interchip_send_buffer.pm_data.heading = gps_data.heading;
+    interchip_send_buffer.pm_data.speed = gps_data.ground_speed;
+    interchip_send_buffer.pm_data.satellites = (char)gps_data.num_satellites;
+    interchip_send_buffer.pm_data.positionFix = (char)gps_data.fix_status;
+    
+    checkForFirstGPSLock();
```

Comment:
As mentioned above, everything below here doesn't involve the GPS, and won't get updated if there's no new GPS data (e.g. the GPS module isn't connected).